sharing_portal/templates/sharing_portal/detail.html

```diff
@@ -23,28 +23,31 @@
         <h2 class="artifactTitle">{{ artifact.title }}</h2>
         <div class="artifactActions">
           {% if editable %}
-          <a class="btn btn-default" href="{% url 'sharing_portal:share' artifact.pk %}">
+          <a class="btn btn-default" href="{% url 'sharing_portal:share' artifact.uuid %}">
```

Comment:
Very nice how easy this template was to update

Reply:
Yes, the templates were fairly smooth